google-cloud-storage/src/main/java/com/google/cloud/storage/StorageClass.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.storage;
+
+/**
+ * Enums for the storage classes.
+ * See https://cloud.google.com/storage/docs/storage-classes for details.
+ */
+public enum StorageClass {
```

Comment:
After put some thought I think enum is better. The benefit of having enum instead of string lower the user effort of constructing the value themselves (enums can autocomplete by IDE), it would be also clearer to know which values are by reading the enum class. Also, it would be a breaking change anyway if storage adds a new type of storage class. Even though we use strings, a new string value can still potentially break users' code.